rtgui/histogrampanel.cc

```diff
@@ -33,7 +33,36 @@ using namespace rtengine;
 //
 //
 // HistogramPanel
-HistogramPanel::HistogramPanel () : panel_listener(nullptr)
+HistogramPanel::HistogramPanel () :
+    pointer_moved_delayed_call(
+        [this](bool validPos, const Glib::ustring &profile, const Glib::ustring &profileW, int r, int g, int b)
+        {
+            bool update_hist_area;
+
+            if (!validPos) {
+                // do something to un-show vertical bars
+                if (histogramRGBArea) {
+                    histogramRGBArea->updateBackBuffer(-1, -1, -1);
+                }
+                update_hist_area = histogramArea->updatePointer(-1, -1, -1);
+            } else {
+                // do something to show vertical bars
+                if (histogramRGBArea) {
+                    histogramRGBArea->updateBackBuffer(r, g, b, profile, profileW);
+                }
+                update_hist_area = histogramArea->updatePointer(r, g, b, profile, profileW);
+            }
+            if (histogramRGBArea) {
+                histogramRGBArea->queue_draw();
+            }
+            if (update_hist_area) {
+                histogramArea->queue_draw();
+            }
+        },
+        50,
```

Comment:
@Floessie Are this hardcoded delays?

Reply:
@heckflosse Yes. Feel free to change them or make them configurable (in `dev`, not on this branch).